main.go

```diff
@@ -170,8 +229,8 @@ func runTests(machine *client.RobotClient) {
 	f2 := initializeFiles("./wheeledData")
 	defer f2.Close()
 
-	f.WriteString("type,linveldes,angveldes,time,posX,posY,theta,power\n")
-	f2.WriteString("type,linveldes,angveldes,time,posX,posY,theta,power\n")
+	f.WriteString("type,linveldes,angveldes,time,posX,posY,theta\n")
+	f2.WriteString("type,linveldes,angveldes,time,posX,posY,theta\n")
```

Comment:
maybe we can make this a single string so its easier to modify all 6 spots the string gets used?